src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -67,21 +67,21 @@ class AnnualIncomeSection extends React.Component {
           <h6>Spouse</h6>
           <ErrorableTextInput
               errorMessage={getErrorMessage(this.props.data.spouseGrossIncome, message)}
-              label="Spouse Gross Income"
+              label="Gross Annual Income from Employment"
```

Comment:
I feel like it's still helpful to preface each of the labels with `Veteran`, `Spouse`, or `Child`. So it would read: `Spouse Gross Annual Income from Employment`. Just because otherwise the labels are all the same, and, especially from an accessibility perspective, that could be confusing.